solutions/usaco-737.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: usaco-737
+source: USACO Bronze 2017 US Open
+title: Modern Art
+author: Juheon Rhee
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_art_bronze_open17.html)
+
+## Implementation
+**Time Complexity**: $\mathcal O(N^3)$
+
+<LanguageSection>
+
+<PySection>
+
+```py
+WIDTH = 10
+
+res = [[0] * (WIDTH + 1) for _ in range(WIDTH + 1)]
+left, right, down, up = [0] * WIDTH, [0] * WIDTH, [0] * WIDTH, [0] * WIDTH
+
+ans = [0] * WIDTH
```

Comment:
1. rename to `valid_start`
2. use bools instead of ints

Reply:
yeah but then i cant sum it

Reply:
i just ran this on a mac
```py
Python 3.8.9 (default, Jul 19 2021, 09:37:32) 
[Clang 13.0.0 (clang-1300.0.27.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> x = [True, False]
>>> sum(x)
1
>>> 
```

Reply:
oh nice i'll change it then